source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -291,8 +357,46 @@ content: |
              status: 0,
              masterKey: { provider: 'local' }
            }
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+
```

Comment:
should this be a new file or just added into the script previous?

Reply:
* Switched this to a function snippet so this can be run as a standalone file or part of a monolithic script for this whole page